x-pack/plugins/cases/public/containers/messages_storage.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MessagesStorage } from './messages_storage';
+
+describe('useLocalStorage', () => {
+  const storage = new MessagesStorage();
+  beforeEach(() => {
+    localStorage.clear();
```

Comment:
I think this line won't work as intended since the state is kept in a module level binding: `let instance: MessagesStorage;`

Reply:
It should as `MessagesStorage` doesn't maintain its own state. It's just a wrapper around `localStorage` where the state is stored. Maybe I'm missing something?